### PR TITLE
Import the Panel component from the library-simplified-reusable-compo…

### PR DESCRIPTION
**Pull request by adriana-alter**
…nents package

### CODE REVIEW

File: src/stylesheets/app.scss [hunk 1]
```diff
@@ -1,5 +1,5 @@
 @import "../../node_modules/bootstrap/dist/css/bootstrap.css";
-@import "../../node_modules/opds-web-client/lib/stylesheets/app.scss";
```
Comment (EdwinGuzman) [inline src/stylesheets/app.scss]: Removing `panel.scss` was fine but why did you remove this file? It's still needed... and the new file does not replace this.

File: src/stylesheets/app.scss [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
+@import "../../node_modules/library-simplified-reusable-components/lib/stylesheets/app.scss";
```
Comment (EdwinGuzman) [inline src/stylesheets/app.scss]: Testing this does not work. It doesn't work because that's not the name of the file.
Screen Shot 2019-03-26 at 5 17 29 PM

The name of the compiled css is `reusable-components.css` (https://github.com/NYPL-Simplified/reusable-components/blob/master/webpack.prod.config.js#L25) and it should be retrieved from the `dist` folder since that's the webpack compiled folder.

Also, a better way to import scss or css in a project with webpack is to use `~` since webpack understands to use the `node_modules` folder and you can remove the relative path:
```
@import "~library-simplified-reusable-components/dist/reusable-components.css";
```
(I had wanted to but never got a chance to update all the previous imports).

File: src/stylesheets/app.scss (hunk 1 above)
Comment (adriana-alter) [reply src/stylesheets/app.scss]: I'll put it back in just in case, but I'm pretty sure this repo isn't actually using anything from it (I think I had just copy/pasted it from circ-web when I was first setting this up)...

File: src/stylesheets/app.scss (hunk 2 above)
Comment (adriana-alter) [reply src/stylesheets/app.scss]: Ah.  That explains it.  Thank you; good catch!

File: src/stylesheets/app.scss (hunk 1 above)
Comment (EdwinGuzman) [reply src/stylesheets/app.scss]: The repo is using a few mixins from that repo, unfortunately. If you remove it and try to build, it'll give you that warning.

**Comment by EdwinGuzman** [comment]
Tests passed with latest published npm package. Good to go now.
